app/policies/template_policy.rb

```diff
@@ -1,76 +1,81 @@
 class TemplatePolicy < ApplicationPolicy
   attr_reader :user, :template
-
-  def initialize(user, template)
-    raise Pundit::NotAuthorizedError, "must be logged in" unless user
+  
+  def initialize(user, template = Template.new)
+    raise Pundit::NotAuthorizedError, _("must be logged in") unless user.is_a?(User)
     @user = user
     @template = template
   end
-
-  ##
-  # Users can modify templates if:
-  #  - They can modify templates
-  #  - The template which they are modifying belongs to their org
-  ##
-
-  def admin_index?
+  
+  def index?
     user.can_modify_templates?
   end
 
-  def admin_template?
-    user.can_modify_templates?  &&  (template.org_id == user.org_id)
+  def new?
+    user.can_modify_templates?
   end
 
-  def admin_customize?
+  def create?
```

Comment:
super-admins without modify_template privleges will be unable to create templates, consider changing line below to:
``` user.can_modify_templates? || user.can_super_admin ```

Reply:
good catch